package.json

```diff
@@ -40,6 +40,8 @@
     "@heroicons/react": "~2.1.1",
     "@mdx-js/mdx": "^3.0.0",
     "@nodevu/core": "~0.1.0",
+    "@orama/highlight": "^0.1.2",
+    "@oramacloud/client": "^1.0.2",
```

Comment:
Also for anything non-runtime we prefer using `~` range. I guess this would apply to highlight and orama/orama, but for the client I assume that would be `^`

Reply:
both `@orama/highlight` and `@oramacloud/client` are runtime. `@orama/orama` is just used for types

Reply:
Then orama/orama `~`